doc/source/development/rfc/rfc108_driver_removal_3_11.rst

```diff
@@ -0,0 +1,160 @@
+.. _rfc-108:
+
+=====================================================================
+RFC 108: Driver removals for GDAL 3.11
+=====================================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2025-02-18
+Status:        Draft
+Target:        GDAL 3.11
+============== =============================================
+
+Summary
+-------
+
+This RFC communicates about the partial or total removal of obsolete drivers
+for GDAL 3.11.
+
+Motivation
+----------
+
+GDAL is an ever growing code base, with probably more than 50% of the code
+base being used by less of 1% of users. This is isn't very sustainable, and quite
+demotivating for current maintainers who know nothing/little about most drivers,
+have no evidence that they are used, and regularly stumble upon them when doing
+whole codebase enhancements.
+The decision of which driver to removal comes from a mix of "intuition" and
+community feedback from
+https://docs.google.com/spreadsheets/d/12yk0p8rRK4rAYO_VwXboatWkiILAjE2Cepu32lsHhM8/edit?gid=0#gid=0
```

Comment:
The community was [solicited for feedback](https://lists.osgeo.org/pipermail/gdal-dev/2025-January/060111.html) about removing _write_ support for certain drivers, with a link to this spreadsheet. The spreadsheet is more broad in its scope and indicates complete removal of certain drivers. The spreadsheet shows Interlis as the only vector driver being considered for removal, but this RFC proposes to remove 5 drivers (of which Interlis is not one?). It's not easy to understand the connection between the community feedback and the list in this RFC.

Reply:
Interlis write support removal has already been removed prior to this RFC, given blessing from his maintainer.

> It's not easy to understand the connection between the community feedback and the list in this RFC.

Don't try too hard to find logic ;-)